packages/waku/src/router/common.ts

```diff
@@ -1,6 +1,7 @@
 export type RouteProps = {
   path: string;
   searchParams: URLSearchParams;
+  hash?: string;
```

Comment:
I originally thought of adding `hash`, but wasn't confident if it worked correctly.
Does it work with `router.push('foo#bar')`?

Also, I want to avoid optional:
```suggestion
  hash: string;
```

Reply:
> Does it work with router.push('foo#bar')?

Yes, its even covered in the integration test case:
https://github.com/dai-shi/waku/pull/746/files#diff-a8dab8847d02ba73c6b66e738495556ddda5d2dd01b70322aae006f7669ea237R31-R33